src/graph/planner/plan/Logic.h

```diff
@@ -147,10 +147,10 @@ class PassThroughNode final : public SingleInputNode {
 };
 
 // This operator is used for getting a named alias from another executed operator.
-class Argument final : public SingleInputNode {
+class Argument final : public PlanNode {
```

Comment:
Why change this?

Reply:
Treat `Argument` as a leaf node and delete the useless `StartNode`.

Reply:
Emm. A little wired, node with one input should inherit it.

Reply:
Just ok.